src/pages/homepage.tsx

```diff
@@ -93,117 +97,219 @@ const HomePage = () => {
-          <Card
-            image={`/images/reuses/memos-elections-${router?.locale}.png`}
-            className="text__center"
-            title={t('homepage:reuses.memos-elections-fr.title')}
-            subtitle={t('homepage:reuses.memos-elections-fr.subtitle')}
-            author={t('homepage:reuses.memos-elections-fr.author')}
-            link="https://www.reset.tech/resources/memos-on-platforms-behaviour-during-the-2022-french-elections/"
-            center={true}
-          ></Card>
-          <Card
-            image="/images/reuses/tosdr.jpg"
-            title={t('homepage:reuses.tosdr.title')}
-            subtitle={t('homepage:reuses.tosdr.subtitle')}
-            author={t('homepage:reuses.tosdr.author')}
-            link="https://tosdr.org"
-            center={true}
-          ></Card>
-          <Card
-            image="/images/reuses/readability.jpg"
-            title={t('homepage:reuses.readability.title')}
-            subtitle={t('homepage:reuses.readability.subtitle')}
-            author={t('homepage:reuses.readability.author')}
-            link="https://disinfo.quaidorsay.fr/en/open-terms-archive/experiments"
-            center={true}
-          ></Card>
-          <Card
-            image="/images/reuses/scripta-manent.jpg"
-            title={t('homepage:reuses.scripta_manent.title')}
-            subtitle={t('homepage:reuses.scripta_manent.subtitle')}
-            author={t('homepage:reuses.scripta_manent.author')}
-            link="https://disinfo.quaidorsay.fr/fr/open-terms-archive/scripta-manent"
-            center={true}
-          ></Card>
-          <Card
-            image="/images/reuses/tosback.jpg"
-            title={t('homepage:reuses.tosback.title')}
-            subtitle={t('homepage:reuses.tosback.subtitle')}
-            author={t('homepage:reuses.tosback.author')}
-            link="https://tosback.org/"
-            center={true}
-          ></Card>
-          <Card
-            image="/images/reuses/new-reuse.jpg"
-            className="text__center"
-            title={t('homepage:reuses.new_reuse.title')}
-            subtitle={t('homepage:reuses.new_reuse.subtitle')}
-            link="mailto:contact@opentermsarchive.org"
-            center={true}
-          ></Card>
-        </CardList>
+
+      {/* Product newsletter */}
+      <Container layout="wide" dark={true} paddingY={false}>
+        <Container gridCols="9" gridGutters="8" flex={true}>
+          <Column width={40} alignX="center" alignY="center">
+            <IconMail size="128" color="var(--colorBlack400)" strokeWidth="1" />
+          </Column>
+          <Column width={60} subtitle={t('homepage:product-newsletter.title')}>
+            <TextContent marginTop={false}>
+              <p className="mt__M">{t('homepage:product-newsletter.desc')}</p>
+              <form
+                id="sib-form"
+                method="POST"
+                action={
+                  router.locale === 'fr'
```

Comment:
This could be in a translation string

Reply:
yes of course https://github.com/ambanum/OpenTermsArchive.org/pull/161/commits/c21ee552277322a83b287d20964f2a8ebe951982 !